src/components/logo/z-logo/index.tsx

```diff
@@ -48,7 +48,7 @@ export class ZLogo {
     return (
       <div>
         {this.link && (
-          <a href={this.link} target={this.targetblank ? "_blank" : undefined}>
+          <a style={{width: `${this.width}px`, height:`${this.height}px`}} href={this.link} target={this.targetblank ? "_blank" : undefined}>
```

Comment:
Why inline style?

Reply:
the width and height properties of the z-logo define the dimensions of the svg (see lines 23 and 24) but not of the link tag that contains it. Since the aforementioned properties can assume any numeric value, and not being able to use css in js, managing the variability in style inline guarantees a precise correspondence in the dimensions of the two elements involved and consequently a better graphic rendering